app/controllers/tasks_controller.rb

```diff
@@ -1,11 +1,48 @@
-# require 'date'
-
 class TasksController < ApplicationController
+  skip_before_filter  :verify_authenticity_token, :only=>[:twilio_action]
+
   def index
     @task = Task.new
     @tasks = Task.order('completed, priority DESC')
   end
 
+  def twilio_action
+    message = params[:Body].split(",")
+    description = message.first
+    priority = message.last
+
+    # byebug
+
+    if description == 'Show'
+      tasks = Task.where(completed: false)
+
+      if tasks.length == 0
+        render xml: "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>
+          <Response>
+            <Message>Todo List is empty</Message>
+          </Response>"
+      else
+        list = tasks.map(&:description)
+        render xml: "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>
+        <Response>
+          <Message>#{list.join(", ")}</Message>
+        </Response>"
+      end
+    else
+      task = Task.new(
+        description: description,
+        priority: priority
+      )
+
+      if task.save
+        render xml: "<?xml version=\"1.0\" encoding=\"UTF-8\" ?>
+          <Response>
+            <Message>#{description} has been added to your todo list</Message>
+          </Response>"  
```

Comment:
I'm seeing a lot of repetition in these XML responses -- can you DRY these up a bit?